Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <signatures>The quires have Ethiopic quire numbers in the inner upper corner of the recto of the first leaf and the
+                                        verso of the last leaf.</signatures>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="17 23">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Regular hand (<foreign xml:lang="de">Große, schöne archaische Hand</foreign>, 
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">45</citedRange></bibl>).
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">45</citedRange></bibl>,
+                                    <bibl><ptr target="bm:Grohmann1915schrift"/><citedRange unit="page">72, 87</citedRange></bibl>,
+                                    and
+                                    <bibl><ptr target="bm:Zettersteen1899Upsala"/><citedRange unit="page">519</citedRange></bibl>,
+                                    date it to the 15th century.</desc>
+                                <date notBefore="1400" notAfter="1499">15th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#86v"/>
+                                    <desc type="DonationNote">Donation note to the monastery of <ref type="ins" corresp="INS1018DabraE"/>.</desc>
+                                    <q xml:lang="gez">
+                                        ዝመጽሐፍ፡ ዘወሀብዎ፡ ለደብረ፡ ኤልግኔ፡
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#86v"/>
+                                    <desc type="GuestText" ><title type="incomplete" ref="LIT2807RepCh91"/>
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39–40</citedRange></bibl>
+                                        analyses this as two texts, one directed to Mary and another directed to Jesus. He reproduces the
+                                        first stanza of the first part and the entire second part.
```

Comment:
If I understand you correctly, this analysis by Löfgren is contrary to Chaîne or your own understand. In that case, it should be marked:
Either: `<bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39–40</citedRange></bibl> analyses this **incorrectly** as two texts, (...)`
or: `<bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39–40</citedRange></bibl> analyses this as two texts (contrary to <bibl><ptr target="bm:Chaine1913Rep"/><citedRange unit="item">91</citedRange></bibl>), (...)`